client/status/status.go

```diff
@@ -22,7 +26,7 @@ type Status any
 // Note: direct assignment may not be compatibility-safe.
 func ErrFromStatus(st Status) error {
 	if err, ok := st.(error); ok {
-		return err
+		return errors.Join(ErrAPI, err)
```

Comment:
Is it related to 4f6a31992e2745dbe78bd73155d674912837f521?

Reply:
yes, it is. Separate commit for code consistency. Joining errors and checking them

Reply:
In general, we want to be able to compile every commit. And in some ways this is achieved here (if you're to use 1.20). RC9 will be completely 1.18-compatible.